### DISCUSSION TITLE
Is it Possible to Change the Text in the Booleans?

### DISCUSSION THREAD

**Discussion by MelodicAlbuild** [strapi/strapi#6399] [Legacy Topics]
I want to change the Boolean text from On and Off to Yes|No on the Admin Panel but after digging through files for a while I can't find a way to, anyone know how?

**Comment by christianguevara** [comment]
Looks like this is the component ([here](https://github.com/strapi/strapi/blob/ef6e68b545b11a5db8d388d8a9d95759822e3ad5/packages/strapi-helper-plugin/lib/src/components/InputToggle/index.js#L53)) you will have to do [customizations](https://strapi.io/documentation/v3.x/guides/custom-admin.html#custom-admin) in the admin panel. 

I think those labels should be localized 🤔 if you think the same feel free to open a feature request :)

**Reply by lauriejim** [reply]
Yes that the thing to do to update the text here.
